src/main/java/com/vinylteam/vinyl/dao/UserPostDao.java

```diff
@@ -0,0 +1,7 @@
+package com.vinylteam.vinyl.dao;
+
+import com.vinylteam.vinyl.entity.UserPost;
+
+public interface UserPostDao {
+    boolean add(UserPost post);
+}
```

Comment:
For future. Please add empty row after interface declaration and before closing it. It is kind of our code style